internal/job.tpl.yml

```diff
@@ -45,10 +45,10 @@ spec:
           while true; do sleep 1; done;
       resources:
         requests:
-          memory: 1Mi
+          memory: 5Mi
           cpu: 50m
         limits:
-          memory: 2Mi
+          memory: 5Mi
```

Comment:
```
> k get pod
quickstart2-elasticsearch-diag   0/1     CreateContainerError   0          26s
```
```
> k describe pod quickstart2-elasticsearch-diag | grep Failed
  Warning  Failed     2m14s (x11 over 4m28s)  kubelet          
           Error: Error response from daemon: Minimum memory limit allowed is 6MB
```
Depending on the container runtime and the k8s version, there seems to be a minimum memory limit allowed, which is 6MB for me (k8s version: v1.21.6-gke.1500).

Reply:
Yes I realised this as well on my environment it was 4Mi. After discussing with you offline I bumped this limit to 16Mi now 🤞